corehq/toggles/__init__.py

```diff
@@ -1110,16 +1110,6 @@ def _ensure_valid_randomness(randomness):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
-SPLIT_SCREEN_CASE_SEARCH = StaticToggle(
```

Comment:
This does not impact the migration?

Reply:
No, because of the way the migration instantiates a `StaticToggle` class. (The migration doesn't load this one. It just instantiates one with the same slug.)